src/locales/fi.json

```diff
@@ -746,14 +750,24 @@
     "invoiceSent": "Tarjous lähetetty",
     "sendInvoice": {
       "title": "Lähetä lasku",
-      "instructions": {
-        "paragraph1": "Lasku lähetetään asiakkaan sähköpostiosoitteeseen ({{email}}) ja siihen merkitään alla oleva eräpäivä.",
-        "paragraph2": "Lähetettyä laskua ei voi enää muuttaa, mutta sen voi perua. Perumisesta lähetetään ilmoitus asiakkaalle"
-      },
       "missingEmail": "Sähköpostiosoite puuttuu",
       "dueDate": "Laskun eräpäivä"
     }
   },
+  "additionalInvoice": {
+    "create": "Luo uusi lisäpalvelulasku",
+    "createTitle": "Luo uusi lisäpalvelulasku",
+    "sendTitle": "Lähetä lisäpalvelulasku",
+    "berthContract": "Venepaikkasopimukset",
+    "berthPlace": "Venepaikka",
+    "contractPeriod": "Sopimuskausi",
+    "addtionalService": "Lisäpalvelut",
+    "createInvoice": "Luo lasku",
+    "dueDate": "Eräpäivä",
```

Comment:
Some of these can be found in `common.terminology`, or could be moved there if they are missing